Assets/Scripts/Utilities/Settings.cs

```diff
@@ -153,65 +179,124 @@ private static string GetSetting(string key)
     private static void SaveSettings()
     {
         // Create an xml document.
-        XmlDocument xmlDocument = new XmlDocument();
+        XmlDocument xDoc = new XmlDocument();
 
         // Create main settings node.
-        XmlNode settingsNode = xmlDocument.CreateElement("Settings");
+        XmlNode settingsNode = xDoc.CreateElement("Settings");
 
         foreach (KeyValuePair<string, string> pair in settingsDict)
         {
             // Create a new element for each pair in the dict.
-            XmlElement settingElement = xmlDocument.CreateElement(pair.Key);
+            XmlElement settingElement = xDoc.CreateElement(pair.Key);
             settingElement.InnerText = pair.Value;
             Debug.ULogChannel("Settings", pair.Key + " : " + pair.Value);
 
             // Add this element inside the Settings element.
             settingsNode.AppendChild(settingElement);
         }
 
-        // Append Settings node to the document.
-        xmlDocument.AppendChild(settingsNode);
-
-        // Get file path of Settings.xml .
-        string filePath = System.IO.Path.Combine(Application.streamingAssetsPath, settingsFilePath);
+        // Apend Settings node to the document.
+        xDoc.AppendChild(settingsNode);
 
-        // Save document.
-        xmlDocument.Save(filePath);
+        // Save the document.
+        try
+        {
+            xDoc.Save(userSettingsFilePath);
+        }
+        catch (Exception e)
+        {
+            Debug.ULogWarningChannel("Settings", "Settings could not be saved to " + userSettingsFilePath);
+            Debug.ULogWarningChannel("Settings", e.Message);
+        }
     }
 
     private static void LoadSettings()
     {
-        // Initilize the settings dict.
+        // Initialize the settings dict.
         settingsDict = new Dictionary<string, string>();
+        string furnitureXmlText;
 
-        // Get file path of Settings.xml and load the text.
-        string filePath = System.IO.Path.Combine(Application.streamingAssetsPath, settingsFilePath);
-        string furnitureXmlText = System.IO.File.ReadAllText(filePath);
+        // Load the settings XML file.
+        // First try the user's private settings file in userSettingsFilePath.
+        // If that doesn't work fall back to defaultSettingsFilePath.
+        // If that doesn't work fall back to the hard coded furnitureXmlText above.
+        if (System.IO.File.Exists(userSettingsFilePath) == false)
+        {
+            Debug.ULogWarningChannel("Settings", "User settings file could not be found at '" + userSettingsFilePath + "'. Falling back to defaults.");
```

Comment:
This cant be a warning, its expected to not exist the first time you start the game. Log it if you want, but not as a problem. :)